Simplenote/src/main/java/com/automattic/simplenote/NotesActivity.java

```diff
@@ -1010,6 +1010,9 @@ public void onConfigurationChanged(Configuration newConfig) {
                 invalidateOptionsMenu();
             } else {
                 if (mNoteListFragment.isHidden()) {
+                    // Go to NoteEditorActivity if the note editing was fullscreen and orientation was switched to portrait
+                    if (mCurrentNote != null)
```

Comment:
Let's add the curly braces to this `if` statement for code style and readability.

Reply:
Done.